4_advanced_usage/35_directive_ordering.txt

```diff
@@ -0,0 +1,12 @@
+=== Directives ordering
+
```

Comment:
Could you add a paragraph about the motivations behind this (avoid having to wait for multiple runs for convergence) and what should be avoided (a configuration that relies on the order to be applicable)?